stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/EventHandlerImpl.kt

```diff
@@ -256,31 +260,19 @@ internal class EventHandlerImpl(
                 }
 
                 is ReactionNewEvent -> {
-                    // get the message, update the reaction data, update the message
-                    // note that we need to use event.reaction and not event.message
-                    // event.message only has a subset of reactions
-                    batch.getCurrentMessage(event.reaction.messageId)?.let {
-                        val updatedMessage = it.apply {
-                            addReaction(event.reaction, domainImpl.currentUser.id == event.user.id)
-                        }
-                        batch.addMessage(updatedMessage)
-                    }
+                    event.message.enrichWithCid(event.cid)
```

Comment:
We might have some issues with pagination if somebody updates very old message with "MessageUpdatedEvent", "ReactionNewEvent", etc and we persist the message from the event to our database.